### PR TITLE
Create ui

### PR DESCRIPTION
**Pull request by okamil2**
s1

This picture is of what the UI looks like!

### CODE REVIEW

File: src/Buttons.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+.key6{
+    background-color: saddlebrown;
+}
+.key7{
+    background-color: darkgoldenrod;
+}
+.key8{
+    background-color: darkred;
+}
```
Comment (JameelJiwani) [inline src/Buttons.css]: Can we make these class names something more meaningful?

File: src/Buttons.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import './Buttons.css';
+import App from "./App";
+
+function Buttons(){
+    return(
+
+        <div className="keys">
+            <div class={"container"}>
+                    <div>
```
Comment (JameelJiwani) [inline src/Buttons.js]: Fix indentation here and remove unnecessary white space within the return() block

File: src/Buttons.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+//             }
+//         }
+//         else{
+//
+//         }
+//
+//     });
+// }
+//
+// var Key = document.getElementsByClassName("Key");
```
Comment (JameelJiwani) [inline src/Buttons.js]: remove comments

File: src/Buttons.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+                        <button className={"key"}>8</button>
+                        <button className={"key"}>9</button>
+                        <button className={"key key7"} >x</button>
+                    </div>
+
+                    <div>
+                        <button className={"key"}>.</button>
+                        <button className={"key"}>0</button>
+                        <button className={"key"}>00</button>
+                        <button className={"key key8"} id={"calculate"}>=</button>
```
Comment (JameelJiwani) [inline src/Buttons.js]: It doesn't look like these id's do anything

File: src/Screen.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import App from "./App";
+import './Screen.css';
+
+
+function Screen(){
+    return(
+    <div className={"output"}>
+    This is the output screen
```
Comment (JameelJiwani) [inline src/Screen.js]: Since this string is going to be modified. Let's move it into a variable

**Review by JameelJiwani — CHANGES_REQUESTED** [review]
Looks good so far! Just need a bit of cleaning up

File: src/Buttons.js (hunk 3 above)
Comment (okamil2) [reply src/Buttons.js]: These comments are for the logic part that will be completed soon. Did you still want me to remove it for now or it is okay to keep them?

Comment (JameelJiwani) [reply src/Buttons.js]: You don't have to but I would recommend it in order to keep the code base cleaner. I like to keep code snippets like this in Microsoft OneNote or `*.txt` files

File: src/logic [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+//             }
+//         }
+//         else{
+//
+//         }
+//
+//     });
+// }
+//
+// var Key = document.getElementsByClassName("Key");
```
Comment (JameelJiwani) [inline src/logic]: delete this

File: src/Screen.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import App from "./App";
+import './Screen.css';
+
+
+let testVariable;
```
Comment (JameelJiwani) [inline src/Screen.js]: one line

Comment (JameelJiwani) [reply src/Screen.js]: also add a better name

**Review by JameelJiwani — CHANGES_REQUESTED** [review]

**Review by JameelJiwani — APPROVED** [review]

**Pull request merged**